push-to-s3-and-invalidate-cloudfront/action.yml

```diff
@@ -0,0 +1,70 @@
+name: Push files to an S3 bucket destination and invalidate the cloudfront cache
+description: |
+  This action will push files to a destination S3 bucket (or a folder within a bucket) and invalidates the cloudfront cache.
+  The action also removes any stale files from the bucket. The action can also delete files/folders from the bucket.
+
+inputs:
+  s3-bucket:
+    description: The name of the S3 bucket
+    required: true
+  build-directory:
+    description: The directory that contains the build files to upload to S3
+    required: true
+  pull-request-number:
+    description: The pull request number for which a preview needs to be generated/updated
+    required: true

```

Comment:
I really think that instead of passing a PR number, the user should pass the exact directory in the bucket they want their files in. That will cater to a lot more usecases than doing it this way

Reply:
Done!